test/demo/contextChannels.inttest.ts

```diff
@@ -47,7 +47,7 @@ async function setupWindows(...channels: (ChannelId|undefined)[]): Promise<Ident
         const channel = channels[i];
         const identity = appIdentities[i + offset];
 
-        await fdc3Remote.open(testManagerIdentity, identity.uuid);
+        await fdc3Remote.open(testManagerIdentity, identity.name);
```

Comment:
As per Bryan's suggestion https://github.com/HadoukenIO/fdc3-service/pull/108#discussion_r293745141 . We should have a quick discussion about this and make it consistent everywhere

Reply:
FDC3 `open` goes by `name` from the Directory rather than `appID`. Though not sure what guarantees we have about that matching `name` or `uuid` in the `Identity` object. In either case, we're kinda abusing the `Identity` object right? We're just doing this for convenience and we get away with it because everything in our test apps matches.

Anyway, @pjbroadbent @spcfran, grab me whenever

Reply:
Discussion helped, makes sense now 😃